.gitignore

```diff
@@ -0,0 +1,21 @@
+# C/C++ generated
+*.a
+*.ax
+*.d
+*.dll
+*.lib
+*.lo
+*.o
+*.os
+*.ox
+*.Plo
+*.so
+# Binutils tmp linker output of the form "stXXXXXX" where "X" is alphanumeric
+st[A-Za-z0-9][A-Za-z0-9][A-Za-z0-9][A-Za-z0-9][A-Za-z0-9][A-Za-z0-9]
+
+# Python generated
+__pycache__/
+*.pyc
+
+# Temporary files.
+*.tmp
```

Comment:
Tentative change, copied a few sections from godotengine/godot, and added .tmp on the bottom, since that was the only other file being generated on my end.